### PR TITLE
Register APIs

### PR DESCRIPTION
**Pull request by gsanchietti**
New APIs:
- register
- unregister
- info

### CODE REVIEW

File: packages/ns-api/README.md [hunk 1]
```diff
@@ -1041,6 +1041,71 @@ Response example:
 }
 ```
 
+## ns.subscription
+
+Manager server subscription for [my.nethesis.it](https://my.nethesis.it) and [my.nethserver.com](https://my.nethserver.com).
```
Comment (edospadoni) [inline packages/ns-api/README.md:1046]:
```suggestion
Manage server subscription for [my.nethesis.it](https://my.nethesis.it) and [my.nethserver.com](https://my.nethserver.com).
```

**Review by edospadoni — CHANGES_REQUESTED** [review]

File: packages/ns-api/README.md [hunk 2]
```diff
@@ -1041,6 +1041,71 @@ Response example:
...
+```
+
+Error response:
+```json
+{"error": "unregister failure"}
+```
+
+### info
+
+Retrive subscription information from remote server:
```
Comment (andre8244) [inline packages/ns-api/README.md:1086]:
```suggestion
Retrieve subscription information from remote server:
```

File: packages/ns-plug/files/subscription-info [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+#
+# Retrive subscription information
```
Comment (andre8244) [inline packages/ns-plug/files/subscription-info:9]:
```suggestion
# Retrieve subscription information
```

**Review by andre8244 — COMMENTED** [review]
Just a couple of typos

**Review by edospadoni — APPROVED** [review]

**Pull request merged**